src/order/sup_indep.lean

```diff
@@ -253,21 +253,51 @@ lemma independent_pempty (t : pempty → α) : independent t.
 lemma independent.pairwise_disjoint : pairwise (disjoint on t) :=
 λ x y h, disjoint_Sup_right (ht x) ⟨y, supr_pos h.symm⟩
 
-lemma independent.mono {ι : Type*} {α : Type*} [complete_lattice α]
+lemma independent.mono
   {s t : ι → α} (hs : independent s) (hst : t ≤ s) :
   independent t :=
 λ i, (hs i).mono (hst i) $ supr₂_mono $ λ j _, hst j
 
 /-- Composing an independent indexed family with an injective function on the index results in
 another indepedendent indexed family. -/
-lemma independent.comp {ι ι' : Sort*} {α : Type*} [complete_lattice α]
```

Comment:
This change made this less general; with it we have `ι : Type*`, but before we had `ι : Sort*`

Reply:
I confess I had no spotted this but just before I cater for indexing over `Prop`: do we actually want this generality?

Reply:
Given `independent` is defined in this generality, we should certainly be this general in cases where all it takes is writing `Sort` instead of `Type` to make the proof go through.